services/relayer/internal/contracts/functioncall/gpact.go

```diff
@@ -0,0 +1,2031 @@
+// Code generated - DO NOT EDIT.
```

Comment:
if this the wrapper for the GPACT CrosschainControl.sol contract?

Reply:
yes